amd_openvx_extensions/amd_rpp/source/kernel_rpp.cpp

```diff
@@ -2597,6 +2632,27 @@ RpptDataType getRpptDataType(vx_enum vxDataType) {
     }
 }
 
+RpptLayout getRpptLayout(vxTensorLayout layout) {
```

Comment:
Rajy is suggesting to have this a map, instead of switch case
https://github.com/ROCm/MIVisionX/pull/1320#discussion_r1598776414

Please remove this function and add a constant map here, and use this map in openvx extensions